Refactor Kiali documentation (#8461)
kyma-project/kyma: docs/kiali/03-01-kiali.md

---
title: Access Kiali
type: Details
---

> Kiali can be easily accessed from the left menu on the [Kyma Console](/components/console/#overview-overview) by clicking on the `Service Mesh` item.
> Once authenticated, the main Kiali dashboard will show a summary of the service mesh status and the left side menu offers different features such as graphs or configuration validation:

Reviewer: ```suggestion
You can easily access Kiali from the [Kyma Console](/components/console/#overview-overview). To do so, click the **Service Mesh**  tab in the menu on the left.
Once you are authenticated, the main Kiali dashboard will show a summary of the Service Mesh status and the left side menu will offer you features such as graphs or configuration validation:
```

> ![Kiali menu item](assets/overview.png)

Reviewer 2: I'm still not convinced to bundle here screenshots as they will outdate so fast.. but if @bszwarc wants them I'm fine :)

> Graphs are very useful to review the topology of the service mesh:

Reviewer: ```suggestion
Use the graphs to review the topology of the Service Mesh:
```

---
title: Access Kiali
type: Details
---

> Kiali can be easily accessed from the left menu on the [Kyma Console](/components/console/#overview-overview) by clicking on the `Service Mesh` item:

Reviewer 2: I'm not sure if it is really relevant to have screenshots how to find the kiali link and how the entry page looks like. Maybe @bszwarc can advice better her. However, interesting might be the entry URL. That it starts with kiali.xxx always.

Author: I removed the console screenshot and kept the kiali default dashboard and graph as examples of how it looks like.
